CHANGELOG.md

```diff
@@ -13,6 +13,131 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
 
+## v6.14.0 (2016-08-23) TAKE ME TO FLAVOR TOWN
+
+Lots of stuff in this release!
+
+- [#3624](https://github.com/babel/babel/pull/3624) A new preset for `es2017`: it includes the 2 previous stage-3 plugins: async/await (via [transform-async-to-generator](http://babeljs.io/docs/plugins/transform-async-to-generator)) and [trailing commas in functions](syntax-trailing-function-commas). (thanks to @bettiolo for the npm package)
+
+```js
+// .babelrc
+{ "presets": ["es2017"] }
+```
+
+- [#3625](https://github.com/babel/babel/pull/3625), [#3673](https://github.com/babel/babel/pull/3673) A new preset called `latest` to include at transforms ES2015+ (currently ES2015, ES2016, ES2017). You can also pass options down to the `es2015` preset.
```

Comment:
`... include at ...` Im not native english, but this sounds funny to me :) But I might be wrong.